test/master-playlist-controller.test.js

```diff
@@ -212,6 +212,50 @@ QUnit.test('resets SegmentLoader when seeking in flash for both in and out of bu
 
   });
 
+QUnit.only('selects lowest bitrate rendition when enableLowInitialPlaylist is set',
+  function(assert) {
+    this.player = createPlayer({ html5: { hls: { enableLowInitialPlaylist: true } } });
+
+    this.player.src({
+      src: 'manifest/master.m3u8',
+      type: 'application/vnd.apple.mpegurl'
+    });
+
+    this.clock.tick(1);
+
+    const masterPlaylistController = this.player.tech_.hls.masterPlaylistController_;
+    const masterPlaylistLoader = masterPlaylistController.masterPlaylistLoader_;
+
+    let numCallsToSelectInitialPlaylistCalls = 0;
+    let numCallsToSelectPlaylist = 0;
+
+    // master
+    this.standardXHRResponse(this.requests.shift());
```

Comment:
These two responses should go after the `sourceopen` `trigger`, also I would expect that the `numCallsToSelectInitialPlaylistCalls` should increment after just the master response